crew

```diff
@@ -5,9 +5,40 @@ require 'uri'
 require 'digest/sha2'
 require 'json'
 require 'fileutils'
+require 'docopt'
+
+doc = <<DOCOPT
+Chromebrw - Package manager for Chrome OS http://skycocker.github.io/chromebrew/
+
+Usage:
+  #{__FILE__} build [-k|--keep] <name>...
+  #{__FILE__} download <name>...
+  #{__FILE__} help [<command>]
+  #{__FILE__} install [-k|--keep|-s|--build-from-source] <name>...
+  #{__FILE__} remove <name>...
+  #{__FILE__} search [-d|--desc] [<name>...]
```

Comment:
So?  Are you asking me to make both `-v|--version` and `-p|--page` available because you don't like `-d|--desc`?  I don't understand you.   Well, how about `-d|--details` or `-v|--verbose`?